sql/mozfun/event_analysis/event_property_to_match_string/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+---
+description: >
+  Given an event property index from an `event_types` table, returns a regular expression
+  that can be combined with an event index regular expression to match corresponding events
+  within an `events_daily` table's `events` string.
```

Comment:
```suggestion
  Given an event property index from an `event_types` table, returns a regular expression
  that can be combined with an event index regular expression to match corresponding events
  within an `events_daily` table's `events` string.
  The regex expression will match events if they are not missing the property at `property_index`.
```

Reply:
> It took my awhile to understand what this UDF is doing. Basically, it's giving us a match string that is true if the property at `property_index` is _not_ missing. Can we call it something different then? Maybe `event_property_index_to_existence_match_string` (or more concisely `property_index_existence_match_string`).

I've renamed it to `event_property_index_to_match_string` (taking a cue from the existing `event_index_to_match_string` UDF), incorporated your clarification in the description, and changed both new `event_property_*_to_match_string` UDFs to generate the entire regular expressions (rather than expecting them to be concatenated with the `event_index_to_match_string` UDF output), which I think makes things slightly less convoluted.

Reply:
Agreed, thank you for clarifying!